libs/utils/decorate.js

```diff
@@ -24,6 +24,21 @@ export function decorateButtons(el, size) {
   }
 }
 
+export function decorateIconStack(el) {
+  const ulElems = el.querySelectorAll(':scope > ul');
+  if (!ulElems.length) return;
+  const stackEl = ulElems[ulElems.length - 1];
+  stackEl.classList.add('icon-stack-area', 'body-s');
+  const items = stackEl.querySelectorAll('li');
+  [...items].forEach((i) => {
+    if (i.childElementCount === 2 && i.children[0].tagName === 'PICTURE' && i.children[1]?.tagName === 'A') {
```

Comment:
I see optional chaining for the second child only, yet you already check `i.childElementCount === 2`, so it might not be mandatory